src/qibolab/platform.py

```diff
@@ -170,6 +169,283 @@ def create_tii_qw5q_gold(runcard, simulation_duration=None, address=None, cloud=
     return platform
 
 
+def create_tii_IQM5q(runcard, descriptor=None):
+    """Create platform using Zurich Instrumetns (Zh) SHFQC, HDAWGs and PQSC.
+
+    Instrument related parameters are hardcoded in ``__init__`` and ``setup``.
+
+    Args:
+        runcard (str): Path to the runcard file.
+        descriptor (str): Instrument setup descriptor.
+            If ``None`` it will attempt to connect to TII whole Zurich instruments setup.
+    """
+    # Create channel objects
+    channels = ChannelMap()
+    # readout
+    channels |= ChannelMap.from_names("L3-31")
+    # feedback
+    channels |= ChannelMap.from_names("L2-7")
+    # drive
+    channels |= ChannelMap.from_names(*(f"L4-{i}" for i in range(15, 20)))
+    # flux qubits
+    channels |= ChannelMap.from_names(*(f"L4-{i}" for i in range(6, 11)))
+    # flux couplers
+    channels |= ChannelMap.from_names(*(f"L4-{i}" for i in range(11, 15)))
+    # TWPA
+    # channels |= ChannelMap.from_names("L3-10")
+
+    # Map controllers to qubit channels
+    # feedback
+    channels["L3-31"].ports = [("device_shfqc", "[QACHANNELS/0/INPUT]")]
+    channels["L3-31"].power_range = 10
+    # readout
+    channels["L2-7"].ports = [("device_shfqc", "[QACHANNELS/0/OUTPUT]")]
+    channels["L2-7"].power_range = -25  # -5 for punchout
+    # drive
+    for i in range(5, 10):
+        channels[f"L4-1{i}"].ports = [("device_shfqc", f"SGCHANNELS/{i-5}/OUTPUT")]
+        channels[f"L4-1{i}"].power_range = -10
+    # channels[f"L4-15"].power_range = 20  # For coupler slim peak
+
+    # flux qubits (CAREFUL WITH THIS !!!)
+    for i in range(6, 11):
+        channels[f"L4-{i}"].ports = [("device_hdawg", f"SIGOUTS/{i-6}")]
+        # channels[f"L4-{i}"].power_range = 0
```

Comment:
Commented code

Reply:
This may be in use, or at least I want to people calibrating to know this is an option